frontend/src/index.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import "bootstrap/dist/css/bootstrap.css";
```

Comment:
Bootstrap has been added in 2 places, here and in the index.html.

Reply:
I removed it from index.js